### PR TITLE
feat(kit): `Message` add component

### PR DESCRIPTION
**Pull request by mdlufy**
Fixes #

### CODE REVIEW

File: projects/demo/src/modules/components/message/examples/2/index.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<span class="container incoming">
+    <span tuiMessage>
+        <span>I'm defaulted 🤘</span>
```
Comment (waterplea) [inline projects/demo/src/modules/components/message/examples/2/index.html:3]: default

File: projects/demo/src/modules/components/message/examples/2/index.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    </span>
+</span>
+
+<span class="container outgoing">
+    <span
+        appearance="custom"
+        tuiMessage
+    >
+        <span>And I'm colored 🤘🏿</span>
```
Comment (waterplea) [inline projects/demo/src/modules/components/message/examples/2/index.html:12]: image

File: projects/demo/src/modules/components/message/examples/2/index.less [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+:host {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    gap: 1rem;
+}
+
+[tuiAppearance][data-appearance='custom'] {
+    background: var(--Brand-Pro, rgba(175, 242, 24, 1));
```
Comment (waterplea) [inline projects/demo/src/modules/components/message/examples/2/index.less:11]: Let's not use proprietary variables

File: projects/demo/src/modules/components/message/examples/4/index.less [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+.container {
+    display: flex;
+    inline-size: 100%;
+}
+
+.incoming {
+    justify-content: flex-start;
+
+    > [tuiMessage] {
+        margin-inline-start: 1rem;
```
Comment (waterplea) [inline projects/demo/src/modules/components/message/examples/4/index.less:27]: I think we can just add padding 1rem to container

File: projects/demo/src/modules/components/message/index.html [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+            [content]="index + 1 | tuiExample"
+            [heading]="example"
+            [id]="example | tuiKebab"
+        />
+    </ng-template>
+
+    <ng-template pageTab>
+        <tui-doc-demo>
+            <span
+                tuiMessage
```
Comment (waterplea) [inline projects/demo/src/modules/components/message/index.html:21]: Do we need icons on message as well, maybe? Ask design team, I believe I saw in Figma that sometimes message has a chevron on the right

File: projects/kit/components/message/message.options.ts [hunk 6]
```diff
@@ -0,0 +1,15 @@
+import type {Provider} from '@angular/core';
+import {tuiCreateToken, tuiProvideOptions} from '@taiga-ui/cdk/utils/miscellaneous';
+import type {TuiAppearanceOptions} from '@taiga-ui/core/directives/appearance';
+
+export const TUI_MESSAGE_DEFAULT_OPTIONS: TuiAppearanceOptions = {
+    appearance: 'neutral',
+};
+
+export const TUI_MESSAGE_OPTIONS = tuiCreateToken(TUI_MESSAGE_DEFAULT_OPTIONS);
```
Comment (waterplea) [inline projects/kit/components/message/message.options.ts:9]: Let's not introduce options for now, if we only have appearance. Just provide it directly.

File: projects/kit/styles/components/message.less [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+    block-size: auto;
+    box-sizing: border-box;
+    inline-size: fit-content;
+    isolation: isolate;
+    white-space: nowrap;
+    text-align: start;
+    font: var(--tui-font-text-ui-m);
+    border-radius: var(--tui-radius-l);
+
+    :host-context(tui-root._mobile) & {
```
Comment (waterplea) [inline projects/kit/styles/components/message.less:29]: Should we maybe introduce mobile specific design using tuiPlatform and inside mobile theme styles?

File: projects/demo/src/modules/components/message/examples/2/index.less (hunk 3 above)
Comment (mdlufy) [reply projects/demo/src/modules/components/message/examples/2/index.less:11]: Removed

File: projects/kit/components/message/message.options.ts (hunk 6 above)
Comment (mdlufy) [reply projects/kit/components/message/message.options.ts:9]: Removed, provided

File: projects/kit/styles/components/message.less (hunk 7 above)
Comment (mdlufy) [reply projects/kit/styles/components/message.less:29]: why tuiPlatofrm? to add example to demo?

**Review by waterplea — APPROVED** [review]

**Review by splincode — APPROVED** [review]

**Review by vladimirpotekhin — APPROVED** [review]

**Pull request merged**